all-spark-notebook/Dockerfile

```diff
@@ -3,8 +3,6 @@ FROM jupyter/minimal-notebook
 
 MAINTAINER Jupyter Project <jupyter@googlegroups.com>
 
-USER root
```

Comment:
None of the Dockerfiles do any kind of USER switching anymore after these changes are applied. However, explicit is better than implicit so I'm fine adding a USER root to the beginning of every Dockerfile just to make it clear.